lib/core/network/models/proposal_details_model.dart

```diff
@@ -8,72 +7,13 @@ import 'package:json_annotation/json_annotation.dart';
 
 part 'proposal_details_model.g.dart';
 
-enum TokenType {
-  utility,
-  voice,
```

Comment:
why was nearly all code removed from this class? We don't need any of this??

Reply:
Yeah, for now, according to the Figma UI update from @luigicarlini1988 (@arsenijesavic) , we don’t need to use these fields. If we find a need for them later, I can look into a new or better approach otherwise I can always revert these changes.